crates/bevy_app/src/app.rs

```diff
@@ -76,22 +76,18 @@ struct SubApp {
     runner: Box<dyn Fn(&mut World, &mut App)>,
 }
 
+/// Sets the app threading model
+#[derive(Copy, Clone)]
+pub enum AppThreading {
+    /// run app single threaded
+    Single,
+    /// run app multi threaded
+    Multi,
+}
+
 impl Default for App {
     fn default() -> Self {
-        let mut app = App::empty();
-        #[cfg(feature = "bevy_reflect")]
-        app.init_resource::<AppTypeRegistry>();
-
-        app.add_default_stages()
-            .add_event::<AppExit>()
-            .add_system_to_stage(CoreStage::Last, World::clear_trackers.exclusive_system());
-
-        #[cfg(feature = "bevy_ci_testing")]
-        {
-            crate::ci_testing::setup_app(&mut app);
-        }
-
-        app
+        App::new_with_threading_option(AppThreading::Multi)
```

Comment:
Idea: What about adding a `Default` to `AppThreading` and calling `AppThreading::default()` here? Just a suggestion